src/Dfc.CourseDirectory.Core/DataStore/Sql/Models/TLevelDefinitionExemplarContent.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Dfc.CourseDirectory.Core.DataStore.Sql.Models
+{
+    public class TLevelDefinitionExemplarContent
```

Comment:
Because it's always queried separately. Bring back all exemplar content every time a TLevelDefinition is used will massively increase the data volume for no reason.